app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionOverlay.kt

```diff
@@ -148,7 +148,7 @@ class TrackingProtectionOverlay(
 
         val toolbar = getToolbar()
         val trackingProtectionIcon: View =
-            toolbar.findViewById(R.id.mozac_browser_toolbar_tracking_protection_indicator)
+            toolbar.findViewById(R.id.mozac_browser_toolbar_security_indicator)
```

Comment:
I think we were using this id to show the tracking protection onbording, I'm not sure if we are going to need it any more, if we are going to also remove the TP icon animations? 

<img width="444" alt="image" src="https://user-images.githubusercontent.com/773158/122118613-392f9280-cdf6-11eb-9788-cb0fcc952287.png">

Reply:
All the following lines are configuring the dialog.

Reply:
Yes, I switched it over from the tracking protection indicator to the security indicator since that was going away. This is another one of those UX questions if we will keep this.

Reply:
But otherwise, I specify the security indicator here to make sure the window is anchored correctly for the time being even though we don't have a tracking protection icon which the popup mentions.

Reply:
👍🏽